src/java/com/wolfssl/provider/jsse/WolfSSLImplementSSLSession.java

```diff
@@ -163,6 +164,18 @@ public WolfSSLImplementSSLSession (WolfSSLSession in,
         creation = new Date();
         accessed = new Date();
 
+        if (host != null) {
```

Comment:
what's the advantage of this over just having:

`this.host = host`

Since if it's null it is still being set to null anyway.

Reply:
Good catch, I just refactored this to always set `this.host = host` and removed the conditional block.